packages/composites/proof-of-reserves/src/endpoint/reserves.ts

```diff
@@ -34,24 +35,61 @@ const inputParameters: InputParameters = {
     type: 'array',
     description: 'An array of addresses to get the balance from, when `protocol` is set to `list`',
   },
+  disableAddressValidation: {
+    required: false,
+    type: 'string',
+    description: 'Gives the option to disable address validation before the balances are fetched.',
+    default: 'false',
+  },
+  disableDuplicateAddressFiltering: {
+    required: false,
+    type: 'string',
+    description:
+      'Gives the option to disabled the filtering of duplicate addresses in a request. ' +
+      'If this is set to `true` and a duplicate address is contained in the request, the balance of that address will be counted twice.',
+    default: 'false',
+  },
 }
 
 export const execute: ExecuteWithConfig<Config> = async (input, context, config) => {
   const validator = new Validator(input, inputParameters, paramOptions)
-
   const jobRunID = validator.validated.jobRunID
```

Comment:
Let's keep the space here to keep it consistent with other adapters.

```suggestion

  const jobRunID = validator.validated.jobRunID
```